docs/3_setup_guide/Delft_Fiat/hydromt_fiat_gui.qmd

```diff
@@ -0,0 +1,229 @@
+---
+title:	Flood Adapt Model Builder - Graphical User Interface
+filters:
+  - lightbox
+lightbox: auto
+execute:
+  echo: false
+jupyter: python3
+---
+
+The Hydromt FIAT Graphical User Interface (GUI), also refered to as **Flood Adapt Model Builder (FIAT)**, facilitates the process of setting up the **Delft FIAT** model, thus making the **Delft FIAT** model creation accessible for users without modeling experience . 
```

Comment:
Add an introduction/overview about the Delft-FIAT model builder in this opening paragraph, and include the diagram at this level. Something like "The Delft-FIAT model-builder allows users to rapidly set up a Delft-FIAT model using a 'quick-build' option (and link to that section here) using default modeling choices, or to build a tailored Delft-FIAT model (link to that section here)".